src/transforms/ppmt.jl

```diff
@@ -0,0 +1,171 @@
+using EmpiricalCDFs
+using Distributions
+using Optim
+
+"""
+    Ppmt()
+
+Applies the projection pursuit multivariate transform to the table. 
+Taken out from the documentation: Friedman, J. H. (1987). 
+Exploratory Projection Pursuit. Journal of the American Statistical 
+Association, 82(397), 249–266. https://doi.org/10.2307/2289161
+
+"""
+
+struct Ppmt
+    table
+    tol::Real
+    perc::Real
+    trunc::Integer
+    sample::Integer
+end
+
+Ppmt(table) = Ppmt(table,1e-5,.9,10,10)
+
+function normal_score_transform(D, ϵ = 1e-5)
+    #Modifies the aleatory variables so they have standard normal cdf.
+    _, q = size(D)
+    Y = Matrix{Float64}(undef,size(D))
+    for i = 1:q #Computing over the q aleatory variables zᵢ
+        cdf = EmpiricalCDF(D[:,i])
+        sort!(cdf)
+        Y[:,i] = [quantile(Normal(), q) for q in cdf(D[:,i]).-ϵ]
+    end
+    return Y
+end
+
+function data_sphering(Y)
+    #Normalize the random variables (send them to the unitary sphere)
+    Σ = cov(Y,dims=1)
+    d, U = eigen(Σ)
+    D = Diagonal(d)
+    return Y*U*(D^(-.5))*U'
+end
+
+function r_transform(Z,α)
+    #Transforms a row of r.v. into a convex combination r.v. with values in [-1,1] #and standard normal distribution.
+    α_unit = (1/norm(α)) .* α .* (norm(α) != 1) + α .* (norm(α) == 1)
+    X = Z * α_unit
+    2 * cdf(Normal(), X) .- 1
+end
```

Comment:
We should probably rename this function to reflect what it actually does. It is converting the input variables into a new scale that the paper calls R. Maybe `rscores(Z, a)` would be a better name?